src/views/setup/Login.tsx

```diff
@@ -17,14 +17,9 @@ export const Login = (props: LoginProps): React.ReactElement => {
   const [translate] = useTranslation()
   const [username, setUsername] = React.useState('')
   const [password, setPassword] = React.useState('')
-  const [error, setError] = React.useState('')
 
   const complete = async (): Promise<void> => {
-    try {
-      await dispatch(apiActions.login(username, password).execute)
-    } catch (e) {
-      setError(e.message)
-    }
+    await dispatch(apiActions.login(username, password).execute)
```

Comment:
When we handle the error globally I suppose this is fine, but I would still catch it here and ignore it, if you throw it (as described in the other comment)